### PR TITLE
clamav - update to version 1.0.0

### PR DESCRIPTION
**Pull request by fritzkink**

### CODE REVIEW

**Comment by AndWac** [comment]
There seem to be missing dependencies on gcc-4-runtime and g++-4-runtime which are needed by rustup:
╰─➤  gmake sample-manifest && gmake test                                                                                                                                2 ↵
/bin/rm -f -r /export/home/andreas/oi-userland/components/sysutils/clamav/clamav-1.0.0
/usr/bin/env RUBY_VERSION=2.3 /export/home/andreas/oi-userland/tools/userland-unpack  /Projekte/OpenIndiana/userland-sources/clamav-1.0.0.tar.gz
/usr/bin/touch /export/home/andreas/oi-userland/components/sysutils/clamav/clamav-1.0.0/.unpacked
(cd /export/home/andreas/oi-userland/components/sysutils/clamav/clamav-1.0.0 && for f in COPYING/*; do cat $f >> COPYING.clamav ; done )
/usr/bin/touch /export/home/andreas/oi-userland/components/sysutils/clamav/clamav-1.0.0/.prep
(/bin/rm -f -rf /export/home/andreas/oi-userland/components/sysutils/clamav/build/amd64 ; /bin/mkdir -p /export/home/andreas/oi-userland/components/sysutils/clamav/build/amd64)
( cd clamav-1.0.0; export RUSTUP_HOME=/export/home/andreas/oi-userland/components/sysutils/clamav/build/.rust_up; export CARGO_HOME=/export/home/andreas/oi-userland/components/sysutils/clamav/build/.cargo; export RUSTUP_INIT_SKIP_PATH_CHECK=yes; curl https://sh.rustup.rs -sSf | bash -s -- -y --no-modify-path; source /export/home/andreas/oi-userland/components/sysutils/clamav/build/.cargo/env; rustup install 1.65.0; rustup default 1.65.0-x86_64-unknown-illumos; rustup show; cargo install --root=/export/home/andreas/oi-userland/components/sysutils/clamav/build/.cargo cbindgen; );
info: downloading installer
ld.so.1: rustup-init: fatal: libgcc_s.so.1: open failed: No such file or directory
main: line 465:  2513 Killed                  "$@"
/bin/bash: line 1: /export/home/andreas/oi-userland/components/sysutils/clamav/build/.cargo/env: No such file or directory

**Comment by fritzkink** [comment]
I added back those missing packages but in the Makefile I tried to explain why I removed them. Maybe we can find a solution for this in the future.

**Comment by AndWac** [comment]
The we should add such links in our gcc-runtime packages with the appropriate mediator settings.

**Comment by AndWac** [comment]
Please rename results-64.master to results-all.master which is the default for some time (or add USE_COMMON_TEST_MASTER= no to the Makefile.

File: components/sysutils/clamav/Makefile [hunk 1]
```diff
@@ -117,6 +119,16 @@ COMPONENT_TEST_TRANSFORMS += \
...
+# currently there is no system/library/g++-4-runtime installed only two links are needed
+# to build the package:
+#
+#       ln -s $(GCC_LIBDIR)/libgcc_s.so.1 /usr/lib/$(MACH64)/libgcc_s.so.1
+#       ln -s $(GCC_LIBDIR)/liblibssp.so.0 /usr/lib/$(MACH64)/libssp.so.0
+#
+# Also some other package can provide this two links (which one?)
+# Basically it proves that the very old gcc is not needed as also mentioned by Joshua
+# M. Clulow
+
```
Comment (AndWac) [inline components/sysutils/clamav/Makefile:131]: Using these links into gcc-7 doesn't quite work as expected:

Test project /export/home/andreas/oi-userland/components/sysutils/clamav/build/amd64
    Start 1: libclamav
1/6 Test #1: libclamav ........................   Passed   18.57 sec
    Start 2: libclamav_rust
2/6 Test #2: libclamav_rust ...................   Passed    0.44 sec
    Start 3: clamscan
3/6 Test #3: clamscan .........................   Passed    9.67 sec
    Start 4: clamd
4/6 Test #4: clamd ............................***Failed  132.11 sec
    Start 5: freshclam
5/6 Test #5: freshclam ........................   Passed    4.70 sec
    Start 6: sigtool
6/6 Test #6: sigtool ..........................   Passed    2.89 sec

83% tests passed, 1 tests failed out of 6

Total Test time (real) = 168.40 sec

The following tests FAILED:
	  4 - clamd (Failed)
Errors while running CTest

This failure is repeatable. I will run more tests with gcc-10 and gcc-11.

**Pull request merged**